__mocks__/@ua/react-native-airship.ts

```diff
@@ -0,0 +1,53 @@
+import type {AirshipContact, AirshipPush, AirshipPushAndroid, AirshipPushIOS, AirshipRoot} from '@ua/react-native-airship';
+import {EventType as EventTypeAirship, iOS as iOSAirship} from '@ua/react-native-airship';
+
+const EventType: Partial<typeof EventTypeAirship> = {
+    NotificationResponse: EventTypeAirship.NotificationResponse,
+    PushReceived: EventTypeAirship.PushReceived,
+};
+
+const iOS: Partial<typeof iOSAirship> = {
+    ForegroundPresentationOption: {
+        Sound: iOSAirship.ForegroundPresentationOption.Sound,
+        Badge: iOSAirship.ForegroundPresentationOption.Badge,
+        Banner: iOSAirship.ForegroundPresentationOption.Banner,
+        List: iOSAirship.ForegroundPresentationOption.List,
+    },
+};
+
+const pushIOS: AirshipPushIOS = jest.fn().mockImplementation(() => ({
```

Comment:
Why do we need these `jest.fn().mockImplementation(..` now?

Reply:
This way was the only way to make mock satisfy types from airship, if you have some better idea how to do it let me know 😄

Reply:
Let's move this way then!